extensions/wikia/AdEngine/js/video/player/ui/dynamicReveal.js

```diff
@@ -0,0 +1,40 @@
+/*global define, setTimeout*/
+define('ext.wikia.adEngine.video.player.ui.dynamicReveal', [
+	'ext.wikia.adEngine.slotTweaker',
+	'wikia.document'
+], function (slotTweaker, doc) {
+	'use strict';
+
+	var videoAspectRatio = 640 / 360;
+
+	function add(video, params) {
+		var slot = doc.getElementById(params.slotName),
+			slotExpanded = false,
+			slotWidth;
+
+		video.addEventListener('start', function () {
+			if (params.isDynamic && !slotExpanded) {
```

Comment:
I think you could extract `params.isDynamic` condition and don't add event listener at all when not required.